sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JacksonGenerator.scala

```diff
@@ -127,7 +153,7 @@ private[sql] class JacksonGenerator(
       (row: SpecializedGetters, ordinal: Int) =>
         val v = row.get(ordinal, dataType)
         sys.error(s"Failed to convert value $v (class of ${v.getClass}}) " +
-          s"with the type of $dataType to JSON.")
+            s"with the type of $dataType to JSON.")
```

Comment:
Why change this?

Reply:
ummm.. I think it maybe modified by IDE automatically.  I'll recover it.